wrapper/telemetry-wrapper.js

```diff
@@ -542,6 +547,28 @@ window.TelemetryWrapper.go = function (params, element) {
     for (var tick of graphEl.querySelectorAll('.mg-extended-y-ticks')) {
       tick.setAttribute('x2', window.parseFloat(tick.getAttribute('x2')) + missingWidth);
     }
+
+    if (hist.kind == 'categorical') {
+      // Rotate the x axis labels
```

Comment:
Please introduce a comment up here that the following code contains magic values that seem to work for now.